src/hotspot/share/opto/superword.cpp

```diff
@@ -1049,30 +1034,37 @@ void SuperWord::set_alignment(Node* s1, Node* s2, int align) {
   }
 }
 
-// Extend packset by following use->def and def->use links from pack members.
-void SuperWord::extend_packset_with_more_pairs_by_following_use_and_def() {
+// Extend pairset by following use->def and def->use links from pair members.
+void SuperWord::extend_pairset_with_more_pairs_by_following_use_and_def() {
```

Comment:
Not really. Inside we need lots of access to the SuperWord components (check `alignment`, `stmts_can_pack`, etc). You could always pass in a `SuperWord` reference, but that is not really nicer.
Maybe it would be easier in the future, once I make some other changes. But we will always need to have other information available (dependency graph, types, etc - these are all VLoopAnalyzer submodules), but for now we also rely on `alignment`, which I plan to remove, and lives in SuperWord.